nlp/voice/index.js

```diff
@@ -180,6 +194,8 @@ router.post('/:locale/voice/tts', iv.validatePOST({ text: 'string' }, { json: tr
 router.post('/@:model_tag/:locale/voice/stt', upload.single('audio'), restSTT);
 router.post('/@:model_tag/:locale/voice/query', upload.single('audio'),
     iv.validatePOST({ metadata: 'string' }, { json: true }), restSTTAndNLU);
-router.post('/@:model_tag/:locale/voice/tts', iv.validatePOST({ text: 'string' }, { json: true }), tts);
+
+// provide TTS API keyed off to :gender, so people can choose the voice gender to be used
+router.post('/@:gender/:locale/voice/tts', iv.validatePOST({ text: 'string' }, { json: true }), tts);
```

Comment:
No. The previous API was correct: the @-sign denotes a model tag. The reason for this API design is that the whole "https://nlp.almond.stanford.edu/@com.foo" can be specified as a NLP server URL in Genie, so all API endpoints that are available at the root must be available also under a specific model tag (even if the model tag doesn't affect the model at all, as in the case of STT/TTS).
Use the body parameters to specify a gender.

Reply:
ok, now I understand why those are there even if unused.